CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 - Major version bump due to dependencies
 - Updated `stylelint` to v14.0.0
+- Add stylelint-config-standard-scss
```

Comment:
Potentially dumb question: Will these styles apply to regular ol' `.css` files? Or is this smart about only applying them to `.scss` files?

Reply:
That's not a dumb question at all! The answer is that the rules will run on every file, but only apply to SCSS things. eg, they look for double-slash comments, and new SCSS-specific at-rules, and SCSS variables.

I've been running a stylelint config that has SCSS bundled for years on vanilla CSS projects and never run into any problems.

Reply:
Great! Thank you 🙂